.github/workflows/helm-checks.yaml

```diff
@@ -124,26 +124,35 @@ jobs:
     runs-on: ubuntu-22.04
     env:
       RELEASE_REGISTRY: "localhost:5000/keptn"
+      DATETIME: ${{ needs.prepare_ci_run.outputs.DATETIME }}
     steps:
       - name: Check out code
         uses: actions/checkout@v3
 
       - name: Generate helm charts
         run: |
-          make helm-package RELEASE_REGISTRY=${{ env.RELEASE_REGISTRY }} TAG=dev-${{ needs.prepare_ci_run.outputs.DATETIME }}
-          sed -i 's/imagePullPolicy: Always/imagePullPolicy: Never/g' ./helm/chart/templates/rendered.yaml
-          sed -i 's/ghcr.keptn.sh\/keptn\/functions-runtime:.*/localhost:5000\/keptn\/functions-runtime:dev-${{ needs.prepare_ci_run.outputs.DATETIME }}/g' ./helm/chart/templates/rendered.yaml        
-          rm -r ./helm/chart/charts
+          make helm-package RELEASE_REGISTRY=${{ env.RELEASE_REGISTRY }} TAG=dev-${{ env.DATETIME }}
+          sed -i 's/functionRunnerImage:ghcr.keptn.sh\/keptn\/functions-runtime:.*/localhost:5000\/keptn\/functionRunnerImage:functions-runtime:dev-${{ env.DATETIME }}/g' ./helm/chart/values.yaml
```

Comment:
do we still need this replacement, since the functions runner image is now being passed using `--set` during the installation?

Reply:
you are right this is done already as set
I will remove it